src/js/base/module/Toolbar.js

```diff
@@ -135,15 +135,15 @@ export default class Toolbar {
   activate(isIncludeCodeview) {
     let $btn = this.$toolbar.find('button');
     if (!isIncludeCodeview) {
-      $btn = $btn.not('.btn-codeview').not('.btn-fullscreen');
+      $btn = $btn.not('.note-keep');
```

Comment:
It's a little bit hard to find the relationship between the classname `.note-keep` and the option name `codeviewButton`. How about to change it to `.note-btn-keep-on-codeview` or similar?

Reply:
I used `.note-keep` to keep it short, it can be anything as long as it works.

Reply:
how about `.note-codeview-keep` to define the relationship and keep it short?

Reply:
This classname will be used only in Summernote, not exposed to users. So I'm OK whether it is long or not.